### PR TITLE
LIT 6834 to 37

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Some new records for Abbadie mss with commentaries; revisions in others.
I will delete  LIT6824CommentariesOldTest in a separate PR.

### CODE REVIEW

File: 4001-5000/LIT4117Interpretation.xml [hunk 1]
```diff
@@ -41,9 +44,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <p/>
+            <p>A collection of commentaries on Old Testament books (from Genesis to Ruth) in Gǝʿǝz and Amharic with parts showing great variability, 
+            but also sharing a common core (according to R. Cowley).</p>
```
Comment (CarstenHoffmannMarburg) [inline 4001-5000/LIT4117Interpretation.xml:48]: Would you mind to encode the title of Cowleys publication in a bibl-ref?

Comment (CarstenHoffmannMarburg) [reply 4001-5000/LIT4117Interpretation.xml:48]: I have seen the reference below: bm:Cowley1988Exeget
Do you think, it is redundant in the abstract?

**Review by CarstenHoffmannMarburg — COMMENTED** [review]
Well done!

File: 4001-5000/LIT4117Interpretation.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 4001-5000/LIT4117Interpretation.xml:48]: I do not think we do it regularly; I personally tend not to overfill the short abstract text with references, as they (not numerous) are below anyway.

File: 4001-5000/LIT4117Interpretation.xml [hunk 2]
```diff
@@ -58,35 +65,62 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                    <label>Commentary on Leviticus</label>
+                 </div>
+                 <div type="textpart" subtype="section" xml:lang="gez" xml:id="Numbers">
+                    <label>Commentary on Numbers</label>
+                 </div>
+                 <div type="textpart" subtype="section" xml:lang="gez" xml:id="Deuteronomy">
+                    <label>Commentary on Deuteronomy</label>
+                    
+                 </div>
+                 <div type="textpart" subtype="section" xml:lang="gez" xml:id="Hosea">
```
Comment (eu-genia) [inline 4001-5000/LIT4117Interpretation.xml:113]: Hosea or Joshua?

**Review by eu-genia — APPROVED** [review]

File: 4001-5000/LIT4117Interpretation.xml [hunk 3]
```diff
@@ -61,6 +61,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <revisionDesc>
          <change who="PL" when="2016-11-17">Created XML record from Ethio authority google spreadsheet</change>
          <change who="DR" when="2017-04-01">Added keywords and adjustments</change>
+         <change who="DN" when="2023-06-01">Added internal structure and abstract</change>
```
Comment (CarstenHoffmannMarburg) [inline 4001-5000/LIT4117Interpretation.xml:64]: Just a question for me to learn: "Added internal structure", that means, you added subdivisions and chapters to the work record?

**Pull request merged**